### ISSUE TITLE
consult

### ISSUE THREAD

**Issue by dymolly** [salmanmaq/segmentationNetworks#1]
hello，how to run it

**Comment by ccyy666** [comment]
excuse me ,after I run the code, there is a problem of fuzzy prediction chart. Can you ask me how to solve it?

**Comment by salmanmaq** [comment]
Hi. Can you describe what you mean by the fuzzy prediction chart? If you got the code to work, pull requests and contributions to the repository are welcome too. Unfortunately, I haven't had the time to get this to a good state in a long time.

**Comment by ccyy666** [comment]
My code ability is not very strong，so i want to ask you for some help.After i run the code , i found the prediction image output during training is fuzzy，i‘m sorry that there is no way to put pictures here, so there is no way to show you the output pictures
here the picture, i'm not sure if you could see it
![4B9D072957357059267CF01FFBB109B7](https://user-images.githubusercontent.com/90020234/159148155-f0675a19-e616-4df5-b215-78e9365835b1.png)

**Comment by salmanmaq** [comment]
Sharing the picture is quite appropriate. I can see that something is not quite right. Can you let me know what training epoch this is? How long you have trained for? Can you tell what version of PyTorch, and CUDA you are using? That can have an impact as well. Are you training the model from scratch? This could possibly also be due to training the model from scratch as well. We first pre-trained the model on m2cai-tool dataset, and then fine-tuned it on our own dataset.

**Comment by ccyy666** [comment]
thank you, I solved the problem today. The version of CUDA I use is 0.0.1 and the version of pytorch I use is 1.9.0.I set the epoch to 50 and trained for about 5 hours.This happens when I set the learning rate to 0.00001, but when I adjust the learning rate to 0.05, the predicted value is clear again. What is the reason?
![1647857173(1)](https://user-images.githubusercontent.com/90020234/159240323-f368f6ba-3b94-4ced-b2bb-0d959aa37ef8.png)
n

**Comment by salmanmaq** [comment]
This is great to see. I think the learning rate of 0.00001 is too low, and the network doesn't learn much, even if trained for a relatively long time since the weight updates are very small. When you increase the learning rate to 0.05, the weight updates are larger, and thus the network is able to learn more quickly (and in this case, learn at all). That's why you are able to get better results with the larger learning rate.
